delegate_main.h

```diff
@@ -114,6 +114,7 @@ class Delegate {
   void CreateCacheOp(const OpData& op_data);
 
   std::vector<std::shared_ptr<tim::vx::Operation>>& GetOps() { return ops_; }
+  int GetOperationOutput() { return op_info_.outputs[0]; }
```

Comment:
why always return first output tensor? You can add an index parameter for this api then you can get any output you want.